examples/_utils/plot_dataset.py

```diff
@@ -0,0 +1,246 @@
+    ax2.set_yticks([])
+    plt.colorbar(
+        im, ax=ax2, orientation="horizontal", label="Variable importance", pad=0.5
+    )
+
+    plt.subplots_adjust(top=1.0, bottom=0.2)
+
+
+def plot_pvalue_H0(
+    beta_hat,
+    pvalue,
+    pvalue_corrected,
+    one_minus_pvalue,
+    one_minus_pvalue_corrected,
+    vmin=0.0,
+    vmax=1.0,
+):
+    """
+    Plot for the confidence in the hipothse that the variables are important.
+    Parameters:
+    - beta_hat: Coefficients of the estimated variable of importance (n_features,)
+    - pvalue: pvalue of each variable of importance (n_features,)
+    - pvalue_corrected: corrected pvalue of each variable of importance (n_features,)
+    - one_minus_pvalue: 1 - pvalue of each variable of importance (n_features,)
+    - one_minus_pvalue_corrected: 1 - corrected pvalue of each variable of importance (n_features,)
+    - vmin: Minimum value of the colorbar (float)
+    - vmax: Maximum value of the colorbar (float)
+
+    Returns:
+    - a figure with 3 subplots
+    """
+    fig, (ax1, ax2, ax3) = plt.subplots(3, 1, figsize=(6.4, 4.8))
+    # plot beta_hat
+    im_beta_hat = ax1.imshow(np.expand_dims(beta_hat, 0), vmin=vmin, vmax=vmax)
+    ax1.set_title("beta hat:variable of importance", fontdict={"fontweight": "bold"})
+    ax1.set_yticks([])
+    colbar_beta_hat = plt.colorbar(
+        im_beta_hat,
+        ax=ax1,
+        orientation="horizontal",
+        label="Variable importance",
+        pad=0.2,
+    )
+    colbar_beta_hat.ax.xaxis.labelpad = 0
+    # plot pvalue
+    im_pvalue = ax2.imshow(
+        np.expand_dims(pvalue, 0),
+        norm=LogNorm(vmin=np.min(pvalue), vmax=np.max(pvalue)),
+        cmap=plt.cm.viridis.reversed(),
+    )
+    ax2.set_title(
+        "pvalue of each variable of importance", fontdict={"fontweight": "bold"}
+    )
+    ax2.set_yticks([])
+    colbar_pvalue = plt.colorbar(
+        im_pvalue, ax=ax2, orientation="horizontal", label="pvalue", pad=0.2
+    )
+    colbar_pvalue.ax.xaxis.labelpad = 0
+    # plot pvalue_corrected
+    im_pvalue_corrected = ax3.imshow(
+        np.expand_dims(pvalue_corrected, 0),
+        norm=LogNorm(vmin=np.min(pvalue_corrected), vmax=np.max(pvalue_corrected)),
+        cmap=plt.cm.viridis.reversed(),
+    )
+    ax3.set_title(
+        "corrected pvalue of each variable of importance",
+        fontdict={"fontweight": "bold"},
+    )
+    ax3.set_yticks([])
+    colbar_pvalue_corrected = plt.colorbar(
+        im_pvalue_corrected,
+        ax=ax3,
+        orientation="horizontal",
+        label="corrected pvalue",
+        pad=0.2,
+    )
+    colbar_pvalue_corrected.ax.xaxis.labelpad = 0
+
+    plt.subplots_adjust(top=1.0, hspace=0.3)
+
+
+def plot_pvalue_H1(
+    beta_hat,
+    pvalue,
+    pvalue_corrected,
+    one_minus_pvalue,
+    one_minus_pvalue_corrected,
+    vmin=0.0,
+    vmax=1.0,
+):
+    """
+    Plot for the confidence in the hipothse that the variables are not important.
```

Comment:
I don't quite get the difference between the two functions

Reply:
The first function plot_pvalue_H0 plots just the pvalue and the corrected pvalue with imshow.
The second function plot_pvalue_H1 plots just the 1-pvalue and the 1-corrected pvalue with imshow.
I want it to separate the t-test to be an important variable and not to be an important variable.

It's actually just a basic plot but this needs to be improved for mixing with the score (beta_hat). It should be replaced by a plot with the score of each variable and the associate p-value for being an important variable or not.